src/bundler/bundle_v2.zig

```diff
@@ -1783,13 +1835,15 @@ pub const BundleV2 = struct {
                 this.graph.heap.gc(true);
             }
         }
-        var log = &load.completion.?.log;
+        const log = this.bundler.log;
```

Comment:
Why was this changed? This will crash if the bundler thread writes to `logger.Log` at the same time as the main thread. When using `Bun.build` that can happen.